### PR TITLE
update standalone requirements, settings and instructions for Django 1.8

### PR DESCRIPTION
**Pull request by higs4281**
Upgrade standalone to Django 1.8

## Changes
- Bumps requirements, settings and README setup instructions.

## Testing

- Try following the README for setting up standalone.

## Review

- any of @amymok @mistergone @niqjohnson @marteki 
## Checklist

* [x] CHANGELOG has been updated

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -70,9 +60,9 @@ grunt build
 
 (You can also run `grunt watch` to have the build run automatically after edits to JavaScript source files, HTML templates, and assets in `retirement_api/static/retirement`. `livereload` is turned on if you have a [LiveReload browser extension](http://livereload.com/extensions/).)
 
-Load the app's database tables and content.
+Create a standalone database and Load the app's tables and content.
```
Comment (niqjohnson) [inline README.md]: Lowercase "l" in "Load"

**Comment by niqjohnson** [comment]
All the Python setup instructions are working for me. For some reason, though, `npm install` never finishes and causes my computer to hang indefinitely. I can't imagine any of these changes caused that, though, so if no one else runs into that problem, it's obviously something weird going on with my setup.

Also (and this can be done as a separate PR) the `grunt build` command and the "You can also run `grunt watch`..." paragraph of the readme are outdated now that we're using gulp.

**Comment by higs4281** [comment]
Note to all: I still need to update the standalone header.

**Comment by niqjohnson** [comment]
Not sure what was happening earlier, but npm is working for me now (when in doubt, restart, I guess). Following the new readme instructions gets me a totally functional standalone app. Thanks for nixing those references to grunt, too.

👍

**Comment by marteki** [comment]
Had to `brew upgrade` to get Python- and Django-related package updates, but now? It works wonderfully.

:+1:

**Comment by higs4281** [comment]
@marteki @niqjohnson @mistergone simulated V1 header/footer added for standalone.

**Comment by marteki** [comment]
I'm not getting the navigation menu with the simulated header. Is that intentional?

**Comment by higs4281** [comment]
We may have to go without the navigation menu. can't find what's missing, and it's not exactly relevant for a standalone app.

**Comment by marteki** [comment]
Then :+1: from me!

**Comment by higs4281** [comment]
@marteki sorry, but can you check this one more time? I found the [bug](https://code.djangoproject.com/ticket/24569) that was crashing you, and it was fixed in Django 1.8.1. Standalone settings were specifying 1.8. Now we call for the latest Django 1.8.x

Also, migrations themselves were still trying to import South. I've converted them to native Django migrations.

**Comment by marteki** [comment]
Deleted everything I had set up for the virtual environment and db.sqlite3 file, then started through the README anew. All smooth and good until I loaded the server; `http://localhost:8000/before-you-claim/` gave a URL routing error, but `http://localhost:8000/retirement/before-you-claim/` worked.

Either a routing adjustment, or a README adjustment?

**Comment by marteki** [comment]
HA! You caught it too. 😛

**Pull request merged**